packages/types/src/hub.ts

```diff
@@ -116,6 +116,13 @@ export interface Hub {
    */
   addBreadcrumb(breadcrumb: Breadcrumb, hint?: BreadcrumbHint): void;
 
+  /**
+   * This is the setter for lastEventId.
+   *
+   * @returns The event id which has set.
+   */
+  setLastEventId(id: string): string;
```

Comment:
```suggestion
  setLastEventId?(id: string): string;
```

It has to be optional, as there might be multiple hub versions used, see https://github.com/getsentry/sentry-javascript/pull/4013 as an example of breakage.